lib/ExpensiMark.js

```diff
@@ -284,22 +284,21 @@ export default class ExpensiMark {
             },
             {
                 /**
-                 * Use \b in this case because it will match on words, letters,
-                 * and _: https://www.rexegg.com/regex-boundaries.html#wordboundary
-                 * The !_blank is to prevent the `target="_blank">` section of the
-                 * link replacement from being captured Additionally, something like
-                 * `\b\_([^<>]*?)\_\b` doesn't work because it won't replace
-                 * `_https://www.test.com_`
-                 * Use [\s\S]* instead of .* to match newline
+                 * Ignore <pre>, <code>, <a>, <mentin-user> tag contents first, so _blank is ignored due to it a property of <a>,
```

Comment:
@skyweb331 Can you please only remove those comments that are irrelevant after the change?

Reply:
@sobitneupane Removed the comment

Reply:
@skyweb331 Sorry for the misunderstanding. I mean to say please keep older comments as well if they are still relevant.